pkg/registry/apis/dashboard/register.go

```diff
@@ -642,12 +657,13 @@ func (b *DashboardsAPIBuilder) storageForVersion(
 		return err
 	}
 
-	// Expose read only library panels
-	if libraryPanels != nil {
+	// Expose read library panels
+	//nolint:staticcheck // not yet migrated to OpenFeature
+	if libraryPanels != nil && b.features.IsEnabledGlobally(featuremgmt.FlagGrafanaAPIServerWithExperimentalAPIs) {
```

Comment:
^^^ this is only added on dev instances with this feature toggle

This lets us commit soon and then follow up with:
* better integration tests
* /api -> /apis routing

before we release it in cloud

Reply:
With this change (checking the flag here), aren't we removing the read library panels in prod?
IIRC this was a dependency for gitsync.